core/dbt/include/global_project/macros/materializations/models/materialized_view/create_materialized_view.sql

```diff
@@ -0,0 +1,9 @@
+{% macro get_create_materialized_view_as_sql(relation, sql) -%}
+    {{- log('Applying CREATE to: ' ~ relation) -}}
```

Comment:
Make this debug logging

Reply:
It looks like this is already the lowest level logging:
https://docs.getdbt.com/reference/dbt-jinja-functions/log

Adding `info=True` would also throw it into stdout.